src/audio/AudioManager.cpp

```diff
@@ -231,3 +242,54 @@ void AudioManager::loadEffect(const char *fileName){
 
     chun[fileName] = sound;
 }
+
+/**
+*   Function: fadeMusicOut(int ms)
+*               int ms: amount of milliseconds to fade out music
+*
+*   Returns: void
+*
+*   Date: April 5, 2017
+*
+*   Programmer: Alex Zielinski
+*
+*   Description:
+*       Wrapper fucntion to fade music out. The amount of time is takes to fade
+*       the music out is specified by the paramater int ms (in milliseconds)
+*/
+void AudioManager::fadeMusicOut(int ms){
+    
+    Mix_FadeOutMusic(ms);
+}
+
+/**
+*   Function: playMenuMusic(const char *fileName1, const char *fileName2)
+*               const char *fileName1: first menu music audio file
+*               const char *fileName2: second menu music audio file
+*
+*   Returns: void
+*
+*   Date: April 5, 2017
+*
+*   Programmer: Alex Zielinski
+*
+*   Description:
+*       Randomly selects one of the 2 menu background music audio files
+*       to play when the menu loads
+*/
+void AudioManager::playMenuMusic(const char *fileName1, const char *fileName2){   // generates either 1 or 2
+
+    srand (time(NULL)); // random seed generator
+    int random = rand() % 2 + 1; // randomly select 1 or 2
```

Comment:
Uhm, is there a reason for the need of 1 / 2 instead of just  `rand() % 2` and then use 0 & 1 ?